bindings/python/CMakeLists.txt

```diff
@@ -14,7 +14,25 @@
 
 project(python)
 
-add_subdirectory(third_party)
+execute_process(
```

Comment:
I think for nix, there is no python nanobind package. I think we can do `find_package(nanobind 1.8.0 CONFIG REQUIRED)`, and if `nanobind_FOUND` is not defined we can do the fetch content?